modules/channel-web/src/views/lite/translations/index.tsx

```diff
@@ -17,24 +16,17 @@ import ru from './ru.json'
 import uk from './uk.json'
 
 const defaultLocale = 'en'
-const baseTranslations = { en, fr, pt, es, ar, ru, uk }
-const availableLocale = Object.keys(baseTranslations)
-
-// Handle region language subtags like 'en-US' and 'ar-IQ'
-const isTranslationMatching = prop => key => String(prop).split('-')[0] === key
-const translations = new Proxy(baseTranslations, {
-  get: (target, prop) => target[Object.keys(target).find(isTranslationMatching(prop))]
-})
+const translations = { en, fr, pt, es, ar, ru, uk }
 
-const getUserLocale = (available?: any, defaultFallback?: string) => {
+const getUserLocale = () => {
   const locale = navigator.language || navigator['userLanguage'] || ''
   const langCode = locale.split('-')[0]
 
-  return available && !available.includes(langCode) ? defaultFallback : langCode
+  return Object.keys(translations).includes(langCode) ? langCode : defaultLocale
```

Comment:
Translation is an object with a lookup in constant time why looping overt the keys ? 

`return translations[langCode] || defaultLocale`